code/modules/research/designs/autolathe_desings/autolathe_designs_medical_and_dinnerware.dm

```diff
@@ -206,3 +207,13 @@
 	build_path = /obj/item/reagent_containers/glass/bottle/vial/large
 	category = list("initial","Medical","Medical Designs")
 	departmental_flags = DEPARTMENTAL_FLAG_MEDICAL
+
+
+/datum/design/hypospray/mkii
+	name = "Hypospray Mk. II"
+	id = "hypospray_mkii"
+	build_type = PROTOLATHE | NO_PUBLIC_LATHE
+	materials = list(/datum/material/iron = 1600, /datum/material/glass = 1000)
+	build_path = /obj/item/hypospray/mkii
+	category = list("initial", "Medical","Medical Designs")
+	departmental_flags = DEPARTMENTAL_FLAG_MEDICAL
```

Comment:
These should be locked behind more exotic materials rather than just metal and glass.  couple hundred silver/gold so that while, yes, it is round start, it isn't mass producible without some input from cargo. Given the fact they're pretty damn powerful.